SPECS/ruby/ruby.spec

```diff
@@ -66,12 +66,12 @@
 %global strscan_version         3.0.1
 %global syslog_version          0.1.0
 %global tempfile_version        0.1.2
-%global time_version            0.2.0
+%global time_version            0.2.2
 %global timeout_version         0.2.0
 %global tmpdir_version          0.1.2
 %global tsort_version           0.1.0
 %global un_version              0.2.0
-%global uri_version             0.11.0
+%global uri_version             0.11.1
```

Comment:
I think if we bump this version, there should be a corresponding source package that we should also download to be sure this change is reflected.

Maybe I am missing something trivial, but I think this may not be the right way to address this CVE.
Since we have an explicit provides at line 242, this in my opinion would mean, we changed a number from `0.2.0 -> 0.2.2` without actually doing the upgrade.